.github/workflows/jit.yml

```diff
@@ -74,7 +74,7 @@ jobs:
             runner: windows-latest
           - target: aarch64-pc-windows-msvc/msvc
             architecture: ARM64
-            runner: windows-latest
+            runner: windows-aarch64
```

Comment:
Can we set a default for forks here as well?

It looks like the two windows-aarch64 jobs will be stuck "Waiting for a runner to pick up this job..." forever for a runner that doesn't exist.




<table>
<tr>
<th><a href="https://github.com/python/cpython/actions/runs/14111802620?pr=131559">upstream</a>
<th><a href="https://github.com/hugovk/cpython/actions/runs/14112081000/job/39533783171">fork</a>
<tr>
<td>
<img width="283" alt="image" src="https://github.com/user-attachments/assets/6e667d90-ba22-4fd4-a9d3-9aa506e4b9d9" />
<td>
<img width="297" alt="image" src="https://github.com/user-attachments/assets/147bcd3e-e21c-4400-be20-cc34efecaa6c" />
</table>

Reply:
JIT is now "passing" on forks:

<img width="712" alt="image" src="https://github.com/user-attachments/assets/eb789fa3-6022-48e2-80ea-9fa4b4c1bfc5" />

https://github.com/hugovk/cpython/actions/runs/14125740287/job/39574818114

Quote marks because it's only doing checkout/setup and no build or tests. But maybe that's okay as this is only for the short term until they're available for all. (We could try skipping the checkout/setup too, but again, maybe this is fine for now.)

Reply:
Yes, that's the old trick we used to limit the builds for the linux runners when they were a paid service.